.github/workflows/SDK-ci.yml

```diff
@@ -0,0 +1,33 @@
+name: NodeJS SDK Wrapper CI
+on:
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+env:
+  LT_USERNAME: ${{ secrets.LT_USERNAME }}
+  LT_ACCESS_KEY: ${{ secrets.LT_ACCESS_KEY }}
+  PROJECT_TOKEN: ${{ secrets.PROJECT_TOKEN }}
+  LT_SDK_DEBUG: true
+  SMARTUI_DEBUG: true
+
+jobs:
+  smartui-nodejs-sdk:
+    name: Execute NodeJS sdk build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2

```

Comment:
Here instead of v2, please use v3 checkout
```
steps:
    - name: Checkout code
      if: github.event_name == 'pull_request'
      uses: actions/checkout@v3
      with:
        ref: ${{ github.head_ref }}
  ```